tasks/config/packages.js

```diff
@@ -0,0 +1,22 @@
+export default (grunt) => {
+  const version = grunt.config.get('pkg.version');
+  const productionPath = `kibana/${version.match(/\d\.\d/)[0]}`;
+  const stagingPath = `kibana/staging/${version.match(/\d\.\d\.\d/)[0]}-XXXXXXX/repos/${version.match(/\d\./)[0]}x`;
+  const rpmFolder = 'centos';
+  const debFolder = 'debian';
```

Comment:
Just giving you a heads up: we might have to make some changes to these folder and path names with the unified release process, but we should move forward with this for now.